.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: python
 sudo: false
 cache:
-  apt: true
   directories:
-    - $HOME/.cache/pip
+  - $HOME/.cache/pip
+  - $HOME/.cache/pre-commit
```

Comment:
@guewen should we push this change on cache into https://github.com/OCA/maintainer-quality-tools/pull/613 ?

Reply:
@yvaucher this was from an older configuration I guess.